nfv_ovs_dpdk.md

```diff
@@ -254,11 +254,11 @@ oc kustomize --load-restrictor LoadRestrictionsNone dataplane-operator/examples/
 Custom `OpenStackDataPlaneService` called `nova-custom-ovsdpdk`
 has been created as described in
 the [documentation to configure OpenStack to use OVS DPDK](ovs_dpdk.md).
-The `nova-custom-dpdk` can be seen in the
+The `nova-custom-ovsdpdk` can be seen in the
```

Comment:
Maybe a nit.  Looks like the suffix might be slightly different [1] (`-ovs-dpdk`).

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/c70f4334ddbf7b87a566245e95708feaef64752b/examples/ovs_dpdk/kustomization.yaml#L5

Reply:
done